src/axom/sidre/interface/c_fortran/wrapDataStore.cpp

```diff
@@ -26,7 +26,7 @@ SIDRE_DataStore* SIDRE_DataStore_new(SIDRE_DataStore* SHC_rv)
   // splicer begin class.DataStore.method.new
   axom::sidre::DataStore* SHCXX_rv = new axom::sidre::DataStore();
   SHC_rv->addr = static_cast<void*>(SHCXX_rv);
-  SHC_rv->idtor = 0;
+  SHC_rv->idtor = 1;
```

Comment:
What's `SCH_rv->idtor`?  

Does it make sense for this to have change from 0 to 1?

Reply:
idtor is the index of the destructor. A bug was fixed with https://github.com/LLNL/shroud/commit/6a740252ad3ddf3d9f105ad5830fbaf843a88d72#diff-03302db14fba2604ab029652dfc22804 which changed the owner to "caller" for constructors. Before, when it was 0, the destructor was not being called properly.
The way most people use the Fortran wrapper, the constructor is not called directly from Fortran. Instead it is called from C++ then the address of the object is stuffed into the Fortran derived type where owner is set to "library" i.e. it will not be destroyed from Fortran.

Reply:
Thanks @ltaylor16 ! Just checking to make sure there were no inadvertent changes.